src/abstractdatatable/abstractdatatable.jl

```diff
@@ -793,6 +798,146 @@ function Base.hash(dt::AbstractDataTable)
     return @compat UInt(h)
 end
 
+"""
+Converts null-free `Nullable` columns to their non-`Nullable` equivalent. Modifies `dt` in-place.
+
+
+```julia
+denullify!(dt::AbstractDataTable)
+```
+
+**Examples**
+
+```julia
+dt = DataTable(A = NullableArray(1:3), B = [Nullable(i) for i=1:3])
+denullify!(dt)
+dt
+```
+
+"""
```

Comment:
The preferred format for docstrings is
```julia
"""
    denullify!(dt::AbstractDataTable)

Convert `NullableArray` columns that do not contain null values to a non-`Nullable`
equivalent array type. The table `dt` is modified in place.

# Examples

```jldoctest
julia> dt = DataTable(A = NullableArray(1:3), B = [Nullable(i) for i = 1:3])
<the output as it would appear in the REPL>

julia> denullify!(dt)
<the output as it would appear in the REPL>
``
"""
```
The function signature is at the top, indented by four spaces rather than inside a triple-backquote markdown code block. Then the functionality is described beneath that using the imperative, e.g. "convert" rather than "converts." Sections are given using markdown section markers, i.e. multiples of `#`, and it's generally a good idea to use `jldoctest` for examples in docstrings.